src/main.rs

```diff
@@ -20,49 +31,83 @@ fn main() -> Result<()> {
         .filter(|note| included_scopes.contains(&note.meta.scope.as_str()))
         .collect();
 
+    println!("{}", "## Source".bold().yellow());
     let source_notes = all_notes
         .iter()
         .filter(|note| note.meta.r#type == ZettelType::Source)
-        .map(|note| concat_source_note(note, &all_notes))
-        .map(save_to_file)
+        .map(|note| {
+            let note = concat_source_note(note, &all_notes);
+            println!();
+
+            note
+        })
+        .map(|note| save_to_file(note, &output_path))
         .count();
 
+    println!("{}", "## Main".bold().yellow());
```

Comment:
Does anyone know how to do this without (passing something that could be a `&'static str` as a variable) or (manually inserting escape codes)?